ironfish/src/network/peerNetwork.ts

```diff
@@ -1368,7 +1368,7 @@ export class PeerNetwork {
 
       // Sync every transaction to the wallet, since senders and recipients may want to know
       // about pending transactions even if they're not accepted to the mempool.
-      await this.node.wallet.syncTransaction(transaction, {})
+      await this.node.wallet.addPendingTransaction(transaction, null)
```

Comment:
Get rid of this second parameter, and always use chain.head.sequence no matter what.